src/wrapper/jnienv.rs

```diff
@@ -830,7 +817,8 @@ impl<'a> JNIEnv<'a> {
 
         let class = self.auto_local(self.get_object_class(obj)?.into());
 
-        unsafe { self.call_method_unsafe(obj, (&class, name, sig), parsed.ret, args) }
+        // TODO come up with solution about methods naming/joining (used to be unsafe)
```

Comment:
Yep, probably good idea, I will check

Reply:
Okay, here is the results. I made a few tests for `call_method_unchecked`:
1. null as object - **Just Error**
2. method id from another class - **SIGSEGV**
3. method id from nullptr - **SIGABRT** 
4. method id from garbage - **SIGSEGV**
5. invalid return type - **Just Error**
6. invalid arguments - **SIGSEGV**.

Reply:
So I think "unchecked" is a good suffix for such methods

Reply:
Great, thanks, I believe there aren't many tests to contribute as most of them crash the process :-)

@DarkEld3r , @alexander-irbis , @jrobsonchase , @skletsun , Which naming would you prefer for methods, accepting a field or method id — `foo_unchecked` or `foo_by_id` (e.g., `new_object_unchecked` vs `new_object_by_id`)? Please consider the discussion in the original issue #121 and this thread.

Reply:
I prefer `foo_unchecked` because even if JVM performs checks they should be used carefully. `_unsafe` suffix isn't appropriate here, so `_unchecked` will do.

Reply:
I'd support @DarkEld3r and vote for __unchecked_.

Reply:
Good, I agree with unchecked. Let's rename the remaining method and move on with this PR. If anyone has better suggestions regarding naming — please open a discussion, as this won't be released immediately.